### PR TITLE
Made some settings' comments clearer

### PR DESCRIPTION
**Pull request by squidgyberries**
Describe your PR, what does it fix/add?

Fixes some possible clarity issues in the comments in the example config file

Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

Is it ready for merging, or does it need work?

Ready for merging

### CODE REVIEW

File: example/hyprland.conf [hunk 1]
```diff
@@ -27,7 +27,7 @@ general {
     col.active_border=0x66ee1111
     col.inactive_border=0x66333333
 
-    apply_sens_to_raw=0 # do not apply the sensitivity to raw input (e.g. used by games where you aim)
+    apply_sens_to_raw=0 # apply the sensitivity to raw input (e.g. used by games where you aim using your mouse)
```
Comment (fufexan) [inline example/hyprland.conf:30]:
```suggestion
    apply_sens_to_raw=0 # whether to apply the sensitivity to raw input (e.g. used by games where you aim using your mouse)
```
this seems more descriptive to me

File: example/hyprland.conf [hunk 2]
```diff
@@ -18,7 +18,7 @@ input {
 
 general {
     max_fps=60 # deprecated, unused
-    sensitivity=0.25
+    sensitivity=0.25 # sensitivity for mouse cursor
```
Comment (fufexan) [inline example/hyprland.conf:21]:
```suggestion
    sensitivity=0.25 # for mouse cursor
```

File: example/hyprland.conf [hunk 3]
```diff
@@ -18,7 +18,7 @@ input {
 
 general {
     max_fps=60 # deprecated, unused
```
Comment (fufexan) [inline example/hyprland.conf:20]: @vaxerski shouldn't we just remove this if it's unused?

File: example/hyprland.conf (hunk 1 above)
Comment (squidgyberries) [reply example/hyprland.conf:30]: yup seems more descriptive to me as well

File: example/hyprland.conf (hunk 3 above)
Comment (squidgyberries) [reply example/hyprland.conf:20]: yeah, seems kind of unnecessary

**Comment by vaxerski** [comment]
maybe we should default sens to 1?

**Comment by fufexan** [comment]
Seems like a good idea. I exclusively change the DPI on my mouse instead of sensitivity.

**Pull request merged**